tests/test_query.py

```diff
@@ -369,27 +352,24 @@ def test_check_user_resource_paths_username(client):
         {
             "policy_id": "test-policy",
             "resource_path": "/a",
-            "permissions": ["temp", "temp-reader"],
             "status": config["DRAFT_STATUSES"][0],
             "should_match": False,
         },
         {
             "policy_id": "test-policy",
             "resource_path": "/a",
-            "permissions": ["temp", "temp-reader"],
             "status": config["FINAL_STATUSES"][0],
             "should_match": False,
         },
         {
             "policy_id": "test-policy",
             "resource_path": "/a",
-            "permissions": ["temp", "temp-reader"],
             "status": "INTERMEDIATE_STATUS",
             "should_match": True,
         },
     ],
 )
-def test_check_user_resource_paths_status(client, list_policies_patcher, test_data):
```

Comment:
`test_check_user_resource_paths_status` disappeared but the `# create a request with the status to test` comment was moved to `test_check_permissions_mismatch`, and there are 2 `test_check_permissions_mismatch` 🤔

Reply:
Aah! My bad! I've accidentally renamed this method to the new method